libraries/AP_Mission/AP_Mission.h

```diff
@@ -415,6 +415,18 @@ class AP_Mission
         // comparison operator (relies on all bytes in the structure even if they may not be used)
         bool operator ==(const Mission_Command &b) const { return (memcmp(this, &b, sizeof(Mission_Command)) == 0); }
         bool operator !=(const Mission_Command &b) const { return !operator==(b); }
+
+        /*
+          special handling for loiter turns from cmd.p1 and type_specific_bits
+         */
+        float loiter_turns(void) const {
```

Comment:
I wonder if it should return bool and the turns as a reference. That would allow checking that the command is actually loiter turns.

Also, maybe `get_loiter_turns`.

Reply:
maybe get_loiter_turns_count() or num_rotations() or something to describe what you're getting back.